<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 5 Spatial scale of control and mixing predicts dynamics of canine rabies | dissertation.utf8</title>
  <meta name="description" content="Dissertation" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 5 Spatial scale of control and mixing predicts dynamics of canine rabies | dissertation.utf8" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://mrajeev08.github.io/dissertation" />
  
  <meta property="og:description" content="Dissertation" />
  <meta name="github-repo" content="mrajeev08/dissertation" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 5 Spatial scale of control and mixing predicts dynamics of canine rabies | dissertation.utf8" />
  
  <meta name="twitter:description" content="Dissertation" />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="modeling-canine-rabies-virus-transmission-dynamics.html"/>
<link rel="next" href="conclusion-1.html"/>
<script src="libs/header-attrs-2.7/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="styles/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Rajeev Dissertation 2021</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#canine-rabies-as-a-test-case"><i class="fa fa-check"></i><b>1.1</b> Canine rabies as a test case</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#prospectus"><i class="fa fa-check"></i><b>1.2</b> Prospectus</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#references"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><i class="fa fa-check"></i><b>2</b> Healthcare utilization, provisioning of post-exposure prophylaxis, and estimation of human rabies burden in Madagascar</a>
<ul>
<li class="chapter" data-level="" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#introduction-1"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#methods"><i class="fa fa-check"></i><b>2.2</b> Methods</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#study-site"><i class="fa fa-check"></i><b>2.2.1</b> Study site</a></li>
<li class="chapter" data-level="2.2.2" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#data-and-analyses"><i class="fa fa-check"></i><b>2.2.2</b> Data and analyses</a></li>
<li class="chapter" data-level="2.2.3" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#ethics-statement"><i class="fa fa-check"></i><b>2.2.3</b> Ethics statement</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#results"><i class="fa fa-check"></i><b>2.3</b> Results</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#pep-provisioning-at-the-clinic"><i class="fa fa-check"></i><b>2.3.1</b> PEP provisioning at the clinic</a></li>
<li class="chapter" data-level="2.3.2" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#rabies-status-and-characteristics-of-biting-animals"><i class="fa fa-check"></i><b>2.3.2</b> Rabies status and characteristics of biting animals</a></li>
<li class="chapter" data-level="2.3.3" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#exposure-status-health-seeking-behavior-and-pep-compliance-of-bite-victims-and-patients-reporting-to-the-armc"><i class="fa fa-check"></i><b>2.3.3</b> Exposure status, health-seeking behavior, and PEP compliance of bite victims and patients reporting to the ARMC</a></li>
<li class="chapter" data-level="2.3.4" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#deaths-averted-and-current-burden-of-human-rabies"><i class="fa fa-check"></i><b>2.3.4</b> Deaths averted and current burden of human rabies</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#discussion"><i class="fa fa-check"></i><b>2.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#key-findings"><i class="fa fa-check"></i><b>2.4.1</b> Key findings</a></li>
<li class="chapter" data-level="2.4.2" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#strengths-and-limitations"><i class="fa fa-check"></i><b>2.4.2</b> Strengths and limitations</a></li>
<li class="chapter" data-level="2.4.3" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#wider-context"><i class="fa fa-check"></i><b>2.4.3</b> Wider context</a></li>
<li class="chapter" data-level="2.4.4" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#conclusions-recommendations"><i class="fa fa-check"></i><b>2.4.4</b> Conclusions &amp; recommendations</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#acknowledgements-1"><i class="fa fa-check"></i><b>2.5</b> Acknowledgements</a></li>
<li class="chapter" data-level="2.6" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#conflict-of-interest-statement"><i class="fa fa-check"></i><b>2.6</b> Conflict of interest statement</a></li>
<li class="chapter" data-level="2.7" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#author-contributions"><i class="fa fa-check"></i><b>2.7</b> Author contributions</a></li>
<li class="chapter" data-level="2.8" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#data-availability"><i class="fa fa-check"></i><b>2.8</b> Data availability</a></li>
<li class="chapter" data-level="2.9" data-path="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html"><a href="healthcare-utilization-provisioning-of-post-exposure-prophylaxis-and-estimation-of-human-rabies-burden-in-madagascar.html#references-1"><i class="fa fa-check"></i><b>2.9</b> References</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><i class="fa fa-check"></i><b>3</b> How geographic access to care shapes disease burden: the current impact of post-exposure prophylaxis and potential for expanded access to prevent human rabies deaths in Madagascar</a>
<ul>
<li class="chapter" data-level="" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#abstract-2"><i class="fa fa-check"></i>Abstract</a>
<ul>
<li class="chapter" data-level="" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#background"><i class="fa fa-check"></i>Background</a></li>
<li class="chapter" data-level="" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#methodology-principal-findings"><i class="fa fa-check"></i>Methodology &amp; Principal Findings</a></li>
<li class="chapter" data-level="" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#conclusions-significance"><i class="fa fa-check"></i>Conclusions &amp; Significance</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#author-summary"><i class="fa fa-check"></i>Author Summary</a></li>
<li class="chapter" data-level="3.1" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#introduction-2"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#methods-1"><i class="fa fa-check"></i><b>3.2</b> Methods</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#estimating-geographic-access-to-pep"><i class="fa fa-check"></i><b>3.2.1</b> Estimating geographic access to PEP</a></li>
<li class="chapter" data-level="3.2.2" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#estimating-bite-incidence"><i class="fa fa-check"></i><b>3.2.2</b> Estimating bite incidence</a></li>
<li class="chapter" data-level="3.2.3" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#modeling-reported-bite-incidence-as-a-function-of-access"><i class="fa fa-check"></i><b>3.2.3</b> Modeling reported bite incidence as a function of access</a></li>
<li class="chapter" data-level="3.2.4" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#modeling-human-rabies-deaths"><i class="fa fa-check"></i><b>3.2.4</b> Modeling human rabies deaths</a></li>
<li class="chapter" data-level="3.2.5" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#estimating-the-impact-of-expanding-pep-provisioning"><i class="fa fa-check"></i><b>3.2.5</b> Estimating the impact of expanding PEP provisioning</a></li>
<li class="chapter" data-level="3.2.6" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#sensitivity-analysis"><i class="fa fa-check"></i><b>3.2.6</b> Sensitivity analysis</a></li>
<li class="chapter" data-level="3.2.7" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#data-and-analyses-1"><i class="fa fa-check"></i><b>3.2.7</b> Data and analyses</a></li>
<li class="chapter" data-level="3.2.8" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#ethics-statement-1"><i class="fa fa-check"></i><b>3.2.8</b> Ethics statement</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#results-1"><i class="fa fa-check"></i><b>3.3</b> Results</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#estimates-of-travel-times-to-clinics-are-high-and-variable-across-madagascar."><i class="fa fa-check"></i><b>3.3.1</b> Estimates of travel times to clinics are high and variable across Madagascar.</a></li>
<li class="chapter" data-level="3.3.2" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#as-travel-times-increase-reported-bite-incidence-decreases."><i class="fa fa-check"></i><b>3.3.2</b> As travel times increase, reported bite incidence decreases.</a></li>
<li class="chapter" data-level="3.3.3" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#current-provisioning-of-pep-substantially-reduces-human-rabies-deaths-but-incidence-of-deaths-remains-high-in-areas-with-poor-access"><i class="fa fa-check"></i><b>3.3.3</b> Current provisioning of PEP substantially reduces human rabies deaths, but incidence of deaths remains high in areas with poor access</a></li>
<li class="chapter" data-level="3.3.4" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#expanding-pep-access-to-underserved-populations-is-effective-at-reducing-human-rabies-deaths-but-this-effect-saturates-as-more-clinics-provision-pep"><i class="fa fa-check"></i><b>3.3.4</b> Expanding PEP access to underserved populations is effective at reducing human rabies deaths, but this effect saturates as more clinics provision PEP</a></li>
<li class="chapter" data-level="3.3.5" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#burden-estimates-are-most-sensitive-to-assumptions-of-underlying-rabies-incidence."><i class="fa fa-check"></i><b>3.3.5</b> Burden estimates are most sensitive to assumptions of underlying rabies incidence.</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#discussion-1"><i class="fa fa-check"></i><b>3.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#main-findings"><i class="fa fa-check"></i><b>3.4.1</b> Main findings</a></li>
<li class="chapter" data-level="3.4.2" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#limitations"><i class="fa fa-check"></i><b>3.4.2</b> Limitations</a></li>
<li class="chapter" data-level="3.4.3" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#broader-context"><i class="fa fa-check"></i><b>3.4.3</b> Broader context</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#conclusion"><i class="fa fa-check"></i><b>3.5</b> Conclusion</a></li>
<li class="chapter" data-level="3.6" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#acknowledgements-2"><i class="fa fa-check"></i><b>3.6</b> Acknowledgements</a></li>
<li class="chapter" data-level="3.7" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#supplementary-appendices"><i class="fa fa-check"></i><b>3.7</b> Supplementary Appendices</a></li>
<li class="chapter" data-level="3.8" data-path="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html"><a href="how-geographic-access-to-care-shapes-disease-burden-the-current-impact-of-post-exposure-prophylaxis-and-potential-for-expanded-access-to-prevent-human-rabies-deaths-in-madagascar.html#references-2"><i class="fa fa-check"></i><b>3.8</b> References</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html"><i class="fa fa-check"></i><b>4</b> Modeling canine rabies virus transmission dynamics</a>
<ul>
<li class="chapter" data-level="" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#introduction-3"><i class="fa fa-check"></i><b>4.1</b> Introduction</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#history-of-modeling-rabies-virus-transmission-dynamics"><i class="fa fa-check"></i><b>4.1.1</b> History of modeling rabies virus transmission dynamics</a></li>
<li class="chapter" data-level="4.1.2" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#the-modeling-backbone-for-canine-rabies"><i class="fa fa-check"></i><b>4.1.2</b> The modeling backbone for canine rabies</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#how-to-model-rabies-virus-transmission"><i class="fa fa-check"></i><b>4.2</b> How to model rabies virus transmission?</a></li>
<li class="chapter" data-level="4.3" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#existing-modeling-studies"><i class="fa fa-check"></i><b>4.3</b> Existing Modeling Studies</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#insights-and-limitations"><i class="fa fa-check"></i><b>4.3.1</b> Insights and limitations</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#the-gap-between-models-and-data"><i class="fa fa-check"></i><b>4.4</b> The gap between models and data</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#bite-data"><i class="fa fa-check"></i><b>4.4.1</b> Bite data</a></li>
<li class="chapter" data-level="4.4.2" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#laboratory-confirmed-case-data"><i class="fa fa-check"></i><b>4.4.2</b> Laboratory confirmed case data</a></li>
<li class="chapter" data-level="4.4.3" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#sequence-data"><i class="fa fa-check"></i><b>4.4.3</b> Sequence data</a></li>
<li class="chapter" data-level="4.4.4" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#dog-population-and-vaccination-data"><i class="fa fa-check"></i><b>4.4.4</b> Dog population and vaccination data</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#conclusions"><i class="fa fa-check"></i><b>4.5</b> Conclusions</a></li>
<li class="chapter" data-level="4.6" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#data-and-code-availability"><i class="fa fa-check"></i><b>4.6</b> Data and code availability</a></li>
<li class="chapter" data-level="4.7" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#references-3"><i class="fa fa-check"></i><b>4.7</b> References</a></li>
<li class="chapter" data-level="4.8" data-path="modeling-canine-rabies-virus-transmission-dynamics.html"><a href="modeling-canine-rabies-virus-transmission-dynamics.html#supplementary-figures"><i class="fa fa-check"></i><b>4.8</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><i class="fa fa-check"></i><b>5</b> Spatial scale of control and mixing predicts dynamics of canine rabies</a>
<ul>
<li class="chapter" data-level="" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#abstract-4"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="5.1" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#introduction-4"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#methods-2"><i class="fa fa-check"></i><b>5.2</b> Methods</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#study-area"><i class="fa fa-check"></i><b>5.2.1</b> Study area</a></li>
<li class="chapter" data-level="5.2.2" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#a-spatially-explicit-individual-based-model-of-rabies-transmission"><i class="fa fa-check"></i><b>5.2.2</b> A spatially-explicit, individual-based model of rabies transmission</a></li>
<li class="chapter" data-level="5.2.3" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#model-selection-parameter-estimation"><i class="fa fa-check"></i><b>5.2.3</b> Model selection &amp; parameter estimation</a></li>
<li class="chapter" data-level="5.2.4" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#assessing-simulated-vaccination-campaigns-impact-on-transmission"><i class="fa fa-check"></i><b>5.2.4</b> Assessing simulated vaccination campaigns impact on transmission</a></li>
<li class="chapter" data-level="5.2.5" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#assessing-the-impact-of-additional-interventions-on-transmission-and-outbreak-probabilities."><i class="fa fa-check"></i><b>5.2.5</b> Assessing the impact of additional interventions on transmission and outbreak probabilities.</a></li>
<li class="chapter" data-level="5.2.6" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#data-and-ethics-statement"><i class="fa fa-check"></i><b>5.2.6</b> Data and ethics statement</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#results-2"><i class="fa fa-check"></i><b>5.3</b> Results</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#the-scale-of-mixing-and-specification-of-introductions-differentiate-models."><i class="fa fa-check"></i><b>5.3.1</b> The scale of mixing and specification of introductions differentiate models.</a></li>
<li class="chapter" data-level="5.3.2" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#capturing-the-spatial-scale-of-vaccination-and-of-mixing-is-key-to-recovering-the-observed-trajectory-of-rabies-in-the-serengeti-district."><i class="fa fa-check"></i><b>5.3.2</b> Capturing the spatial scale of vaccination and of mixing is key to recovering the observed trajectory of rabies in the Serengeti District.</a></li>
<li class="chapter" data-level="5.3.3" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#spatial-coverage-of-vaccination-predicts-control-outcomes-and-interventions-that-target-key-features-of-transmission-improve-these-outcomes."><i class="fa fa-check"></i><b>5.3.3</b> Spatial coverage of vaccination predicts control outcomes and interventions that target key features of transmission improve these outcomes.</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#discussion-2"><i class="fa fa-check"></i><b>5.4</b> Discussion</a></li>
<li class="chapter" data-level="5.5" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#references-4"><i class="fa fa-check"></i><b>5.5</b> References</a></li>
<li class="chapter" data-level="5.6" data-path="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#supplementary-figures-1"><i class="fa fa-check"></i><b>5.6</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="conclusion-1.html"><a href="conclusion-1.html"><i class="fa fa-check"></i><b>6</b> Conclusion</a>
<ul>
<li class="chapter" data-level="6.1" data-path="conclusion-1.html"><a href="conclusion-1.html#future-directions"><i class="fa fa-check"></i><b>6.1</b> Future directions</a></li>
<li class="chapter" data-level="6.2" data-path="conclusion-1.html"><a href="conclusion-1.html#policy-reccommendations"><i class="fa fa-check"></i><b>6.2</b> Policy Reccommendations</a></li>
<li class="chapter" data-level="6.3" data-path="conclusion-1.html"><a href="conclusion-1.html#references-5"><i class="fa fa-check"></i><b>6.3</b> References</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="appendix-additional-publications-and-software-resulting-from-dissertation-work.html"><a href="appendix-additional-publications-and-software-resulting-from-dissertation-work.html"><i class="fa fa-check"></i>Appendix: Additional publications and software resulting from dissertation work</a>
<ul>
<li class="chapter" data-level="" data-path="appendix-additional-publications-and-software-resulting-from-dissertation-work.html"><a href="appendix-additional-publications-and-software-resulting-from-dissertation-work.html#additional-publications"><i class="fa fa-check"></i>Additional publications</a></li>
<li class="chapter" data-level="" data-path="appendix-additional-publications-and-software-resulting-from-dissertation-work.html"><a href="appendix-additional-publications-and-software-resulting-from-dissertation-work.html#additional-software"><i class="fa fa-check"></i>Additional software</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><div class="line-block">DATA AND MODELING TO INFORM CANINE RABIES CONTROL AND ELIMINATION<br />
</div></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies" class="section level1" number="5">
<h1><span class="header-section-number">Chapter 5</span> Spatial scale of control and mixing predicts dynamics of canine rabies</h1>
<div id="abstract-4" class="section level2 unnumbered">
<h2>Abstract</h2>
<p>Canine rabies is a devastating and preventable disease that causes a high burden of deaths globally each year, particularly in low-and-middle countries where investment in control is limited. Given the mounting evidence that elimination of these deaths is possible, the WHO has set a target to eliminate canine rabies deaths in humans by the year 2030. Mathematical models are a powerful tool to inform these efforts, but to date, efforts to develop robust modeling frameworks for domestic dog populations have been hampered by a lack of data, and inconsistencies between model results and observed dynamics. Here, we use a spatially rich, long-term dataset on canine rabies cases in the Serengeti District in Tanzania. Using parallel data on vaccination campaigns and the underlying spatial distribution of the domestic dog population, we use a simulation-based inference approach to fit our models to key spatiotemporal features of the data. We find that incorporating both the spatial scale of population mixing and the scale of control are key to recovering observed dynamics. We use these results to explore different vaccination scenarios, and find that high spatial coverage of vaccination campaigns is necessary to prevent outbreaks of canine rabies. Moving forward these results provide practical insights into control strategies and modeling methods, as well as shed light on key processes driving rabies transmission.</p>
</div>
<div id="introduction-4" class="section level2" number="5.1">
<h2><span class="header-section-number">5.1</span> Introduction</h2>
<p>Rabies is a devastating and fatal disease that causes an estimated 60,000 human deaths annually across the globe <span class="citation">(<a href="introduction.html#ref-hampson2015" role="doc-biblioref">Hampson et al. 2015</a>)</span>. However, these deaths are preventable through post-exposure vaccination, which is highly effective at preventing death if administered promptly after exposure, and through vaccination of domestic dogs, which with sufficient coverage can interrupt transmission and lead to elimination <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-lankester2014" role="doc-biblioref">Lankester et al. 2014</a>)</span>. The World Health Organization and its partners have set a goal to eliminate deaths due to canine-mediated rabies by the year 2030 through a combination of these interventions <span class="citation">(<a href="introduction.html#ref-abela-ridder2016" role="doc-biblioref">Abela-Ridder et al. 2016</a>)</span>.</p>
<p>While there is a rich history of modeling wildlife rabies transmission dynamics, the body of work for canine rabies is much sparser. Critically, traditional models of wildlife rabies dynamics fail to capture patterns of incidence observed in endemic and epidemic scenarios for dog rabies (i.e. low and persistent endemic incidence without substantial depletion of the host population). However, we have a strong understanding of rabies epidemiology and biology: transmission does not appear to scale with dog densities evidenced by consistent <span class="math inline">\(R_{0}\)</span> estimates between 1 - 2 across a range of settings; the infectious period is short and infection is almost invariably fatal; and key epidemiological parameters such as the dispersal kernel and the incubation period are characterized by a long tail. While R0 estimates are consistent and low, there is considerable heterogeneity in transmission with many infectious dogs biting no others, while some may bite upwards of 20 - 50 others <span class="citation">(<a href="introduction.html#ref-rajeev2020modeling" role="doc-biblioref">Rajeev, Metcalf, and Hampson 2020</a>)</span>.</p>
<p>While the epidemiological backbone of rabies is clear, there is little data on disease dynamics in domestic dogs with which to confront models <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-scott2017" role="doc-biblioref">Scott et al. 2017</a>)</span>. In fact, most modeling studies of canine rabies are simulation based, with few studies actually fitting models to data <span class="citation">(<a href="introduction.html#ref-rajeev2020modeling" role="doc-biblioref">Rajeev, Metcalf, and Hampson 2020</a>)</span>. This is largely due to poor surveillance (likely substantially less than 5% of cases are detected annually in endemic settings<span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-townsend2013a" role="doc-biblioref">Sunny E. Townsend et al. 2013</a>)</span>), but given the epidemiological features of rabies combined with low detection rates, inference can be challenging. Rabies outbreaks are highly stochastic and likely driven by heterogeneities and rare events. Therefore traditional inference methods based on times series data alone are likely to fail when confronted with the noisy reality of rabies data.</p>
<p>Recent advances in simulation based inference have facilitated fitting to data from highly stochastic and noisy systems using more complex models with analytically intractable likelihoods <span class="citation">(<a href="introduction.html#ref-funk2020" role="doc-biblioref">Funk and King 2020</a>; <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-hazelbag2020" role="doc-biblioref">Hazelbag et al. 2020</a>)</span>. One such method is Approximate Bayesian Computation, with fits to features of the data rather than each individual observation <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-hazelbag2020" role="doc-biblioref">Hazelbag et al. 2020</a>)</span>. These methods are useful tools for model comparison, and present a way forward to assess what level of complexity is needed in models to predict observed dynamics.</p>
<p>Here we use a uniquely comprehensive long-term dataset on canine rabies cases in space and time across the Serengeti District in Tanzania, where ongoing vaccination campaigns have shaped dynamics over the past twenty years. We build on recent work that highlights the role of spatial structure in mixing <span class="citation">(<a href="introduction.html#ref-Mancyinprep" role="doc-biblioref">Mancy et al., n.d.</a>)</span>, simplifying the model, extending it to consider different movement scenarios, and comparing models at the scale of population mixing (~ 1 km) vs. at the scale of control (village scale). Coupled with data on vaccination and the spatial distribution of the dog population, we build an individual-based spatially explicit model, and using a simulation-based inference approach, fit it to key spatiotemporal features of the observed data to identify models that best fit our data and generate realistic dynamics in endemic situations. Finally, we simulate vaccination campaigns and control strategies based on our findings to inform control efforts.</p>
</div>
<div id="methods-2" class="section level2" number="5.2">
<h2><span class="header-section-number">5.2</span> Methods</h2>
<div id="study-area" class="section level3" number="5.2.1">
<h3><span class="header-section-number">5.2.1</span> Study area</h3>
<p>The Serengeti District in Tanzania is approximately 2000 km<sup>2</sup> in area and is comprised of of multi-ethnic agropastoralist communities (estimated human population of 2.49^{5} at the time of the last census in 2012) with high dog ownership (estimated human-to-dog ratios ranging from 3 to 9). Rabies has been endemically circulating in the district, with documented outbreaks reported as early as the 1950s <span class="citation">(<a href="introduction.html#ref-brunker2015" role="doc-biblioref">Brunker et al. 2015</a>)</span>, and rabies vaccination campaigns have been ongoing in the district since the early 1990s. Through multi-institutional partnerships with the Tanzanian government and vaccine manufacturers, these have expanded into a large scale control program.</p>
<p>Since 2002, a long-term contact tracing study of canine rabies and domestic dogs has been ongoing in the district <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-Hampson2009" role="doc-biblioref">Hampson et al. 2009b</a>)</span>, with data collected on clinical and laboratory diagnosed cases of animal rabies, as well as vaccination and dog ownership data. Here we use three key datasets from this study system:</p>
<ol style="list-style-type: decimal">
<li><p>Case histories of clinical and laboratory diagnosed dog rabies cases between 2002 - 2020 (N = 3179 observations with and approximate date of symptom onset and GPS location (<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:data">5.1</a>A and B).</p></li>
<li><ol start="3" style="list-style-type: decimal">
<li>A full household and dog census of the district conducted between 2009 - 2014, with the spatial location of households, the number of people and of adult dogs and puppies in each household, and their vaccination status (<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:data">5.1</a>C).</li>
</ol></li>
<li><p>A full household and dog census of the district conducted between 2009 - 2014, with the spatial location of households, the number of adult dogs and puppies in each household, the number of individuals in these households, and dog vaccination status (<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:data">5.1</a>D).</p></li>
</ol>
<p>We also used census data on human population sizes from the 2002 and 2012 national census in Tanzania to estimate population growth over the district. Village (i.e. administrative level 3) boundaries for the district were constructed using the distribution of household locations belonging to each village from the dog census data (<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:data">5.1</a>D, <span class="citation">(<a href="introduction.html#ref-Mancyinprep" role="doc-biblioref">Mancy et al., n.d.</a>)</span>).</p>
<div class="figure"><span id="fig:data"></span>
<img src="figs/ch4/fig_data.jpeg" alt="Datasets used in the analyses. A) Shows the time series of monthly rabies cases in domestic dogs and B) shows their spatial distribution across the district. C) The annual number of dogs vaccinated each year during mass campaigns. D) The location of household census data with colors showing the number of dogs owned per household. Polygons delineate village boundaries in the district." width="90%" />
<p class="caption">
Figure 5.1: Datasets used in the analyses. A) Shows the time series of monthly rabies cases in domestic dogs and B) shows their spatial distribution across the district. C) The annual number of dogs vaccinated each year during mass campaigns. D) The location of household census data with colors showing the number of dogs owned per household. Polygons delineate village boundaries in the district.
</p>
</div>

</div>
<div id="a-spatially-explicit-individual-based-model-of-rabies-transmission" class="section level3" number="5.2.2">
<h3><span class="header-section-number">5.2.2</span> A spatially-explicit, individual-based model of rabies transmission</h3>
<p>We developed a discrete-time, individual-based, Susceptible-Exposed-Infectious-Vaccinated transmission model to simulate rabies dynamics in the domestic dog population. Transmission occurs at the individual level with secondary cases drawn from a right censored negative binomial distribution with mean <span class="math inline">\(R_{0}\)</span> and dispersion parameter <span class="math inline">\(k\)</span> (to capture heterogeneity, but constrained so that secondary cases from a single individual do not exceed 100 cases, a realistic range of transmission potential). Movement of infectious individuals occurs in continuous space, with distances either drawn as a dispersal distance (log normal distribution, i.e. case movements are all drawn in reference to the progenitor location) or as a random walk with a Weibull step length distribution, (i.e. case movements are sequential). The timing of when a dog becomes infectious after exposure is drawn from a lognormal serial interval distribution (the infectious period + the incubation period). Previous work has shown the importance of introductions of rabies cases from outside the district in maintaining transmission within the district <span class="citation">(<a href="introduction.html#ref-Mancyinprep" role="doc-biblioref">Mancy et al., n.d.</a>)</span>. We use two approaches to simulate introductions:</p>
<ol style="list-style-type: decimal">
<li>Using inferred locations of introductions from transmission tree reconstruction between 2002 - 2015 to explicitly seed transmission chains</li>
<li>Drawing the number of weekly introductions as a poisson distributed variable (<span class="math inline">\(\iota\)</span>) and randomly seeding them in space</li>
</ol>
<p>Demographic processes occur sequentially and are aggregated to the spatial scale being modeled: new dogs are born according to village level birth rates; dogs die at a fixed rate across the district, and finally vaccination occurs. Vaccinations are allocated from the village level to the spatial scale being modeled, allocated by population size in a given location, and we assume that surviving vaccinated dogs are re-vaccinated in subsequent campaigns, giving a conservative estimate of vaccination. To get initial dog population sizes in each location, we estimate human:dog ratios from the dog census data, and apply them to human population estimates from national census data at the start of our simulation period (2002). We allocate them spatially according to the distribution of households and dogs from the census data. To get village level growth rates, we estimate human population growth between 2002 - 2012 from the national census data at the village level and assume that dog populations grow at that same rate (and assume a fixed death rate, Table <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#tab:tbinputs">5.1</a> <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-czupryna2016" role="doc-biblioref">Czupryna et al. 2016</a>)</span>). We do not apply demographic transitions to exposed or infectious individuals (i.e. they only move out of their classes when moving from exposed to infectious or infectious to removed).</p>
<p>We use a weekly time step to aggregate both demographic and transmission events (i.e. infectious dogs in location x at week t all compete for the same contacts) and explore different scales of spatial mixing: at an 1 km<sup>2</sup> resolution and at the village scale (in UTM Zone 36S such that distances between grid cells are as close to 1 km as possible). Table <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#tab:tbinputs">5.1</a> lists all parameters and their sources. In addition to comparing mixing at the village and 1 km scales, we also explore how further constraining movement of rabid dogs impacts model inference. At the baseline, we do not constrain movements of rabid dogs: that is rabid dog movements to locations outside of the district or to inhabited patches will result in a failed transmission event. We compare this baseline (which we refer to hereafter as the no limits model) with a model that restrict movements to only inhabited patches (i.e. populated patches at the beginning of the simulation, the patch limits model), and finally a model that restricts movement to both uninhabited patches and to the district (the full limits model). For the constrained models, we do not resample individual movements that fail as this could potentially skew the underlying parameter distance distribution (for example, if resampling happens often enough, this can shift both the average and skew of the distribution). Instead, for each movement we draw distances, find grid cells within that distance, and sample movement to valid locations given the model constraints. We assume that infection is completely fatal, and infectious individuals are removed at the end of the weekly timestep during which they became infectious. Finally, to account for imperfect case detection we use a monthly beta binomial detection probability (Table <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#tab:tbinputs">5.1</a>).</p>
<table>
<caption><span id="tab:tbinputs">Table 5.1: </span>Table of parameter inputs and sources.</caption>
<colgroup>
<col width="17%" />
<col width="49%" />
<col width="32%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Input (units)</th>
<th align="left">Distribution</th>
<th align="left">Source</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Secondary case distribution (number of dogs)</td>
<td align="left">Right censored negative binomial distribution with mean = <span class="math inline">\(R_{0}\)</span> and dispersion parameter <span class="math inline">\(k\)</span></td>
<td align="left"><span class="math inline">\(R_{0}\)</span> and <span class="math inline">\(k\)</span> are estimated, prior centered around estimates from <span class="citation"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-Hampson2009" role="doc-biblioref">Hampson et al.</a> (<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-Hampson2009" role="doc-biblioref">2009b</a>)</span></td>
</tr>
<tr class="even">
<td align="left">Weekly rate of introductions</td>
<td align="left">Poisson distribution</td>
<td align="left">Estimated</td>
</tr>
<tr class="odd">
<td align="left">Serial interval distribution (days)</td>
<td align="left">Lognormal distribution with mean = 2.95, and sd = 0.86, for a mean interval of approximately 27 days</td>
<td align="left"><span class="citation"><a href="introduction.html#ref-Mancyinprep" role="doc-biblioref">Mancy et al.</a> (<a href="introduction.html#ref-Mancyinprep" role="doc-biblioref">n.d.</a>)</span></td>
</tr>
<tr class="even">
<td align="left">Dispersal kernel (meters)</td>
<td align="left">Lognormal distribution with mean = 5.58, and sd = 1.79, for a mean dispersal length of approximately 1483 meters</td>
<td align="left"><span class="citation"><a href="introduction.html#ref-Mancyinprep" role="doc-biblioref">Mancy et al.</a> (<a href="introduction.html#ref-Mancyinprep" role="doc-biblioref">n.d.</a>)</span></td>
</tr>
<tr class="odd">
<td align="left">Step length distribution (meters)</td>
<td align="left">Convolved Weibull distribution with shape = 1.12, and scale = 491.1, for a mean step length of approximately 428.35 meters</td>
<td align="left"><span class="citation"><a href="introduction.html#ref-Mancyinprep" role="doc-biblioref">Mancy et al.</a> (<a href="introduction.html#ref-Mancyinprep" role="doc-biblioref">n.d.</a>)</span></td>
</tr>
<tr class="even">
<td align="left">Detection probability (monthly)</td>
<td align="left">Beta binomial distribution with <span class="math inline">\(\beta\)</span> = 8.9, <span class="math inline">\(\alpha\)</span> = 80.1, for a mean monthly detection probability of approximately 0.9</td>
<td align="left"><span class="citation"><a href="introduction.html#ref-Mancyinprep" role="doc-biblioref">Mancy et al.</a> (<a href="introduction.html#ref-Mancyinprep" role="doc-biblioref">n.d.</a>)</span></td>
</tr>
<tr class="odd">
<td align="left">Annual death rate (dogs/yr)</td>
<td align="left">1/25.8 yrs</td>
<td align="left"><span class="citation"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-czupryna2016" role="doc-biblioref">Czupryna et al.</a> (<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-czupryna2016" role="doc-biblioref">2016</a>)</span></td>
</tr>
<tr class="even">
<td align="left">Rate of waning vaccine immunity (dogs/yr)</td>
<td align="left">1/3 yrs</td>
<td align="left"><span class="citation"><a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-Hampson2009" role="doc-biblioref">Hampson et al.</a> (<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-Hampson2009" role="doc-biblioref">2009b</a>)</span></td>
</tr>
</tbody>
</table>



</div>
<div id="model-selection-parameter-estimation" class="section level3" number="5.2.3">
<h3><span class="header-section-number">5.2.3</span> Model selection &amp; parameter estimation</h3>
<p>We use Random Forest Approximate Bayesian Computation (RF-ABC) implemented in the R package <code>abcrf</code> <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-pudlo2015" role="doc-biblioref">Pudlo et al. 2015</a>; <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-raynal2018" role="doc-biblioref">Raynal et al. 2018</a>)</span> to perform model selection and parameter inference. We estimate three parameters: <span class="math inline">\(R_{0}\)</span> (the mean of the secondary case distribution), <span class="math inline">\(k\)</span> (the dispersion parameter of the secondary case distribution), and <span class="math inline">\(\iota\)</span> (the mean weekly number of introductions of rabies cases from outside of the district). We use strong priors based on previous analyses of part of this dataset (Table <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#tab:tbinputs">5.1</a>)<span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-Hampson2009" role="doc-biblioref">Hampson et al. 2009b</a>)</span>. RF-ABC can be used to evaluate both models and potential summary statistics (i.e. key features of the data that the model should capture). Table <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#tab:tbsstats">5.2</a> lists the candidate summary statistics we used for model selection and parameter estimation, encompassing temporal, spatial, and spatiotemporal metrics of the data. RF-ABC ranks each of these parameters by their importance (i.e. Gini importance, specifically how frequently a variable is used for node classification and broadly a relative measure of the discriminative value a statistic has for the classification problem) in predicting a given parameter or in differentiating models. For both model selection and parameter estimation, we generate N = 1e5 simulated sets of summary statistics. To manage computational resources, we stop any simulations where the host population has declined to greater than 15% of the starting population, as these simulations generally lead to extinction of domestic dogs and represent unlikely scenarios. We include these simulations in the dataset, but calculate summary statistics only including the data before the simulation was stopped.</p>
<p>We use the default parameter of 500 trees in the random forest for estimation, and also perform a sensitivity analysis with subsets of the full dataset (subsampling to 75% of simulated dataset) to assess whether estimates are stable. We use prior error rates (the error rate of model classification on simulated datasets withheld from each of the classification trees, i.e. out-of-bag simulations) for model selection to assess the stability of model estimates to the number of simulated datasets and the number of trees in the Random Forest <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-raynal2018" role="doc-biblioref">Raynal et al. 2018</a>)</span>. RF-ABC does not allow for joint parameter estimation, however, we approximate joint posteriors by including parameter sets where all estimated parameters were accepted (i.e. all values were assigned non-zero weights by the random forest classification). We also tested the ability of models to see recovered known parameter estimates using the model to predict out-of-sample simulated datasets across the parameter priors.</p>
<p>We perform model selection for models across all combinations and estimate their posterior probabilities (i.e. given our observed data, the proportion of trees in our RF model that predicted a given model). We then simulate from the estimated posterior distributions (N = 1000) and compare this to our observed data. We score simulations using the continuous ranked probability score <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-jordan2019" role="doc-biblioref">Jordan, Krüger, and Lerch 2019</a>)</span>, and for each simulation of the monthly time series, we also generate centrality scores (i.e. how representative is a given simulation of the ensemble, <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-juul2020" role="doc-biblioref">Juul et al. 2020</a>)</span>) and also the root mean squared error (RMSE) against the observed monthly cases. We also simulate an endemic scenario with no vaccination to see if the models can generate realistic dynamics in this context.</p>
<table>
<caption><span id="tab:tbsstats">Table 5.2: </span>Summary statistics used for fitting in Random Forest ABC.</caption>
<colgroup>
<col width="22%" />
<col width="68%" />
<col width="10%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Summary statistic</th>
<th align="left">Description</th>
<th align="left">Type</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Max I</td>
<td align="left">Maximum monthly number of cases</td>
<td align="left">Temporal</td>
</tr>
<tr class="even">
<td align="left">Median I</td>
<td align="left">Median monthly number of cases</td>
<td align="left">Temporal</td>
</tr>
<tr class="odd">
<td align="left">Mean I</td>
<td align="left">Mean monthly number of cases</td>
<td align="left">Temporal</td>
</tr>
<tr class="even">
<td align="left">Temporal KB</td>
<td align="left">Kullback-Leibler Divergence between observed and simulated distribution of cases temporally</td>
<td align="left">Temporal</td>
</tr>
<tr class="odd">
<td align="left">Spatial KB</td>
<td align="left">Kullback-Leibler Divergence between observed and simulated distribution of cases spatially</td>
<td align="left">Spatial</td>
</tr>
<tr class="even">
<td align="left">Mean distance 4 weeks</td>
<td align="left">Mean distance between cases within an 4 week window</td>
<td align="left">Spatiotemporal</td>
</tr>
<tr class="odd">
<td align="left">Mean distance 8 weeks</td>
<td align="left">Mean distance between cases within an 8 week window</td>
<td align="left">Spatiotemporal</td>
</tr>
<tr class="even">
<td align="left">Mean distance 4 weeks normalized</td>
<td align="left">Mean distances of cases within an 4 week window, normalized by the average distance between all cases</td>
<td align="left">Spatiotemporal</td>
</tr>
<tr class="odd">
<td align="left">Mean distance 8 weeks normalized</td>
<td align="left">Mean distances of cases within an 8 week window, normalized by the average distance between all cases</td>
<td align="left">Spatiotemporal</td>
</tr>
<tr class="even">
<td align="left">Spatial RMSE</td>
<td align="left">Root mean squared error between the observed vs. simulated density of case locations</td>
<td align="left">Spatial</td>
</tr>
<tr class="odd">
<td align="left">Spatial loss</td>
<td align="left">Loss value of the absolute difference between observed and simulated cases locations</td>
<td align="left">Spatial</td>
</tr>
<tr class="even">
<td align="left">Temporal RMSE</td>
<td align="left">Root mean squared error between the observed vs. simulated numbers of monthly cases</td>
<td align="left">Spatial</td>
</tr>
<tr class="odd">
<td align="left">Temporal loss</td>
<td align="left">Loss value of the absolute difference between observed and simulated monthly cases</td>
<td align="left">Temporal</td>
</tr>
<tr class="even">
<td align="left">Mean distance all cases</td>
<td align="left">Average distance between all cases</td>
<td align="left">Spatial</td>
</tr>
<tr class="odd">
<td align="left">ACF Monthly 1-6</td>
<td align="left">Autocorrelation values for monthly time series at 1 - 6 month lag</td>
<td align="left">Temporal</td>
</tr>
<tr class="even">
<td align="left">ACF Weekly 1-10</td>
<td align="left">Autocorrelation values for weekly time series at 1 - 10 week lag</td>
<td align="left">Temporal</td>
</tr>
</tbody>
</table>
</div>
<div id="assessing-simulated-vaccination-campaigns-impact-on-transmission" class="section level3" number="5.2.4">
<h3><span class="header-section-number">5.2.4</span> Assessing simulated vaccination campaigns impact on transmission</h3>
<p>To explore how spatial heterogeneities in vaccination coverage could drive outbreak risk, using the model with the highest posterior probability given our observed data (our ‘best’ model), we simulate across a range of campaign scenarios. We vary both the level of coverage achieved during annual campaigns (campaign coverage), and the proportion of villages which are vaccinated during campaigns (spatial coverage), and use our posterior estimates to simulate transmission dynamics. We simulate these campaigns (N = 100 simulations per combination) with random villages being vaccinated with probability x in each year. We simulate a five year endemic burn-in period, and then simulate annual campaigns over ten years. In the last five year period of the simulation, we calculate summary statistics that capture key metrics of control success such as peak and mean incidence, chain size and length, and weeks where transmission exceeds the 95th% of the mean incidence of introduced cases only. We also calculate a connectivity based summary statistic at the village level (c) which equals the sum of squares of the component size of the village graph, where villages are connected if they are adjacent and if coverage at that time step exceeds a threshold (the campaign coverage for a given simulation).</p>
</div>
<div id="assessing-the-impact-of-additional-interventions-on-transmission-and-outbreak-probabilities." class="section level3" number="5.2.5">
<h3><span class="header-section-number">5.2.5</span> Assessing the impact of additional interventions on transmission and outbreak probabilities.</h3>
<p>Using our best model, we explore how additional interventions implemented alongside vaccination campaigns affect outbreak probabilities and transmission. We simulate approximations of two different interventions in combination with the random village campaigns as described above:
1) ‘Chopping the tail’ of the secondary case distribution <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-Kain2020" role="doc-biblioref">Kain et al. 2020</a>)</span>, where we decrease the censoring value as vaccination increases, to approximate how enhanced surveillance and isolation/quarantine/removal of rabid dogs can improve control outcomes
2) Reducing introductions as coverage increases, i.e. approximating coordination of vaccination campaigns across larger spatial scales</p>
<p>Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-conn-constraint">5.18</a> shows the simulated reductions in introduction rates and maximum allowed secondary cases given district level coverage..</p>
</div>
<div id="data-and-ethics-statement" class="section level3" number="5.2.6">
<h3><span class="header-section-number">5.2.6</span> Data and ethics statement</h3>
<p>The data used in these analyses are part of a long term research project in the Serengeti District approved through the University of Glasgow Ethics Board and by the Tanzania Commission for Science and Technology, and National Institute for Medical Research. Anonymized data and code are available at github.com/mrajeev08/dynamicSD (to be archived on Zenodo and made public upon article submission) and the underlying simulation model is available here (github.com/mrajeev08/simrabid). All analyses were done using R <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-R-program" role="doc-biblioref">R Core Team 2020</a>)</span> and the following key packages: abcrf, ranger, data.table, sf, raster, and foreach, <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-R-sf" role="doc-biblioref">Pebesma 2021</a>; <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-R-data.table" role="doc-biblioref">Dowle and Srinivasan 2020</a>; <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-R-ranger" role="doc-biblioref">Wright, Wager, and Probst 2020</a>; <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-R-raster" role="doc-biblioref">Hijmans 2020</a>; <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-R-foreach" role="doc-biblioref">Revolution Analytics and Weston, n.d.</a>; <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-pudlo2015" role="doc-biblioref">Pudlo et al. 2015</a>)</span>.</p>
</div>
</div>
<div id="results-2" class="section level2" number="5.3">
<h2><span class="header-section-number">5.3</span> Results</h2>
<div id="the-scale-of-mixing-and-specification-of-introductions-differentiate-models." class="section level3" number="5.3.1">
<h3><span class="header-section-number">5.3.1</span> The scale of mixing and specification of introductions differentiate models.</h3>
<p>Model predictions largely varied across two axes: whether introductions were estimated or fixed within the model and the scale of mixing (a 1km<sup>2</sup> scale vs. the village scale). The models with kernel based movement and with estimated introduction rates were ranked highest at both scales, Whether movement was constrained (patch limits or full limits model) did not seem to impact model selection (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-mod-ranks">5.7</a>). Summary statistics that best differentiated the models encompassed both spatial and temporal summary statistics, and the linear-discriminant analysis axes (LDAs, decompositions of our summary statistics) generated by the RF ABC model (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:mfig-mods">5.2</a>B). These results were robust to subsampling the datasets (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-mod-vimp">5.8</a>) and out-of-bag prior error rates were relatively stable with inclusion of additional trees in the random forest model (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-mod-err">5.9</a>). However, broadly, all the models and priors failed to consistently recapture the major components of the summary statistics (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:mfig-mods">5.2</a>A) or clearly differentiate between models within scales (a high prior error rate, with datasets withheld from the classification model misclassified approximately <code>r</code>round(prior_err_rate, 2)*100)`% of the time) (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-mod-ranks">5.7</a>).</p>
<div class="figure"><span id="fig:mfig-mods"></span>
<img src="figs/ch4/mfig_mod.jpeg" alt="Model selection using RF ABC. A) The two major LDA axes generated by the random forest model for the best model at the village and 1 km scale, with the LDA values predicted for the observed data shown as triangles. B) The importance ranking of each variable in discriminating between models." width="90%" />
<p class="caption">
Figure 5.2: Model selection using RF ABC. A) The two major LDA axes generated by the random forest model for the best model at the village and 1 km scale, with the LDA values predicted for the observed data shown as triangles. B) The importance ranking of each variable in discriminating between models.
</p>
</div>

</div>
<div id="capturing-the-spatial-scale-of-vaccination-and-of-mixing-is-key-to-recovering-the-observed-trajectory-of-rabies-in-the-serengeti-district." class="section level3" number="5.3.2">
<h3><span class="header-section-number">5.3.2</span> Capturing the spatial scale of vaccination and of mixing is key to recovering the observed trajectory of rabies in the Serengeti District.</h3>
<p>We estimated three parameters: <span class="math inline">\(R_{0}\)</span>, k, and <span class="math inline">\(\iota\)</span> (for models estimating introductions) using these same models and summary statistics. <span class="math inline">\(R_{0}\)</span> was best predicted by temporal summary statistics, while k and <span class="math inline">\(\iota\)</span> were predicted by spatial and spatiotemporal summary statistics (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:mfig-posts-vimp">5.3</a>, Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-posts-vimp">5.10</a>). Estimates of <span class="math inline">\(R_{0}\)</span> were consistent and identifiable across the models and robust to subsampling simulations (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-posts-err">5.12</a>), but varied significantly across scales and introduction scenarios (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-posts-jntvsfull">5.11</a>). When comparing posterior estimates for the best models at each scale, <span class="math inline">\(R_{0}\)</span> estimates were lower in models at the village scale when compared to the 1 km scale (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:mfig-posts-best">5.4</a>, Table <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#tab:tbpars">5.3</a>). Estimates of k and <span class="math inline">\(\iota\)</span> were more diffuse, and when filtering to parameter sets that had been accepted jointly (i.e. had non-zero weights for all three parameters), estimates of the introduction rate became sharper, while k became bimodal/skewed (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-posts-jntvsfull">5.11</a>). When we tested the best models at each scale to see whether simulated datasets which were not included in the model fitting could be recovered, we found that <span class="math inline">\(R_{0}\)</span> and <span class="math inline">\(\iota\)</span> could be recovered robustly across our prior range, but k was largely unidentifiable (<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-par-recov">5.13</a>).</p>
<table>
<caption><span id="tab:tbpars">Table 5.3: </span>Priors and independent and joint posterior parameter estimates (mean with 95% quantiles in parentheses) for the best model at each scale.</caption>
<thead>
<tr class="header">
<th align="left">Scale</th>
<th align="left">Parameter</th>
<th align="left">Prior</th>
<th align="left">Independent posterior</th>
<th align="left">Joint posterior</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">1x1 km</td>
<td align="left"><span class="math inline">\(R_{0}\)</span></td>
<td align="left">1.13 (0.75 - 1.63)</td>
<td align="left">1.14 (0.99 - 1.28)</td>
<td align="left">1.13 (0.97 - 1.27)</td>
</tr>
<tr class="even">
<td align="left">1x1 km</td>
<td align="left"><span class="math inline">\(k\)</span></td>
<td align="left">1.45 (0.48 - 3.44)</td>
<td align="left">3.04 (0.84 - 7.5)</td>
<td align="left">4.12 (1.26 - 7.16)</td>
</tr>
<tr class="odd">
<td align="left">1x1 km</td>
<td align="left"><span class="math inline">\(iota\)</span></td>
<td align="left">1.13 (0.37 - 2.67)</td>
<td align="left">1.8 (0.81 - 4.63)</td>
<td align="left">1.31 (0.85 - 2.14)</td>
</tr>
<tr class="even">
<td align="left">Village</td>
<td align="left"><span class="math inline">\(R_{0}\)</span></td>
<td align="left">1.13 (0.75 - 1.63)</td>
<td align="left">1.02 (0.91 - 1.13)</td>
<td align="left">1.03 (0.92 - 1.11)</td>
</tr>
<tr class="odd">
<td align="left">Village</td>
<td align="left"><span class="math inline">\(k\)</span></td>
<td align="left">1.46 (0.48 - 3.44)</td>
<td align="left">3.43 (0.97 - 8.13)</td>
<td align="left">4.16 (0.78 - 6.28)</td>
</tr>
<tr class="even">
<td align="left">Village</td>
<td align="left"><span class="math inline">\(iota\)</span></td>
<td align="left">1.13 (0.37 - 2.67)</td>
<td align="left">2.16 (1.03 - 4.19)</td>
<td align="left">1.59 (1 - 2.26)</td>
</tr>
</tbody>
</table>
<div class="figure"><span id="fig:mfig-posts-vimp"></span>
<img src="figs/ch4/mfig_posts_vimp.jpeg" alt="Ranking of summary statistics for importance in predicting each parameter. Summary statistics are colored by the type of statistic (either spatial, temporal, or spatial temporal). Results are shown for the best fitting model at the village and 1 km scale." width="100%" />
<p class="caption">
Figure 5.3: Ranking of summary statistics for importance in predicting each parameter. Summary statistics are colored by the type of statistic (either spatial, temporal, or spatial temporal). Results are shown for the best fitting model at the village and 1 km scale.
</p>
</div>

<div class="figure"><span id="fig:mfig-posts-best"></span>
<img src="figs/ch4/mfig_posts_best.jpeg" alt="Joint posterior estimates compared to prior for the best fitting model at the village and 1 km scale for the three parameters, with prior density in grey." width="90%" />
<p class="caption">
Figure 5.4: Joint posterior estimates compared to prior for the best fitting model at the village and 1 km scale for the three parameters, with prior density in grey.
</p>
</div>

<p>When simulating the data from the posteriors, the models ranked highest by the RF ABC method estimates result in simulations closest to the data observed (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:mfig-sims-best">5.5</a>, Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-sims-full">5.14</a>). Sampling from the joint parameter sets also improved fits to the data (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-sims-joint">5.15</a>, Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-sims-crps">5.16</a>). At both the village and 1 km scale, simulations encompass the trajectory of cases we observed. However, the village scale model fails to capture stable dynamics in an endemic scenario with no vaccination, with transmission resulting in significant declines in the host population, while models at the 1km scale generate stable endemic incidence without depleting the host population (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-sims-endemic">5.17</a>).</p>
<div class="figure"><span id="fig:mfig-sims-best"></span>
<img src="figs/ch4/mfig_sims_best.jpeg" alt="Simulations from the joint posterior distributions of the best fitting model. The grey envelope shows the range of simulations, and the black line is the time series of observed monthly cases. The orange lines show the top three simulations that best fit the data (lowest RMSE) and the purple lines show the top three simulations that have the highest centrality score." width="90%" />
<p class="caption">
Figure 5.5: Simulations from the joint posterior distributions of the best fitting model. The grey envelope shows the range of simulations, and the black line is the time series of observed monthly cases. The orange lines show the top three simulations that best fit the data (lowest RMSE) and the purple lines show the top three simulations that have the highest centrality score.
</p>
</div>

</div>
<div id="spatial-coverage-of-vaccination-predicts-control-outcomes-and-interventions-that-target-key-features-of-transmission-improve-these-outcomes." class="section level3" number="5.3.3">
<h3><span class="header-section-number">5.3.3</span> Spatial coverage of vaccination predicts control outcomes and interventions that target key features of transmission improve these outcomes.</h3>
<p>We find that control outcomes are strongly predicted by both campaign (i.e. the coverage level achieved by a village campaign) and spatial coverage (the proportion of villages covered by a campaign)(Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-base-mets">5.19</a>). We find that campaigns that achieve an annual campaign coverage of 80% of the population at the village scale and a spatial coverage of 80% of villages vaccinated are most likely to prevent outbreaks of canine rabies (defined as greater than 4 months with weekly cases greater than the 95% of the average of the introduced cases) (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:mfig-int-comp">5.6</a>). These broadly track with other metrics of control outcomes as well, such as the average peak size and length of transmission chains, and mean and peak incidence (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-base-mets">5.19</a>). In general, both spatial and campaign coverage are more predictive of peak rather than average metrics (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-base-mets">5.19</a>). Connectivity of vaccinated villages varies the most at intermediate spatial coverage, and this metric also broadly predicts control outcomes within a given campaign scenario (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-conn-mets">5.22</a>).</p>
<div class="figure"><span id="fig:mfig-int-comp"></span>
<img src="figs/ch4/mfig_int_comp.jpeg" alt="Outbreak probabilities (defined here as whether the number of consecutive weeks with cases above the 95th% of the incursion rate exceeding 16 weeks) in relation to a range of campaign coverage and spatial coverage under the baseline scenario and with the additional limits on transmission heterogeneity and introductions." width="90%" />
<p class="caption">
Figure 5.6: Outbreak probabilities (defined here as whether the number of consecutive weeks with cases above the 95th% of the incursion rate exceeding 16 weeks) in relation to a range of campaign coverage and spatial coverage under the baseline scenario and with the additional limits on transmission heterogeneity and introductions.
</p>
</div>

<p>Two additional control strategies, one increasingly censoring transmission as coverage increases, approximating control strategies such as tracing and quarantine, and the second reducing the introduction rate as coverage is increased, approximating coordinated control efforts across larger geographic scales, also greatly improve control outcomes (Figs <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-incs-mets">5.21</a> and <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:sfig-sspread-mets">5.20</a>). In particular, reducing the introduction rate shifts both the spatial coverage and campaign coverage required to limit outbreaks (Fig <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#fig:mfig-int-comp">5.6</a>).</p>
</div>
</div>
<div id="discussion-2" class="section level2" number="5.4">
<h2><span class="header-section-number">5.4</span> Discussion</h2>
<!-- Don't forget to come back to this and include discussion of fitting to stochastic world and a trajectory in observed space rather than a time series per se -->
<!-- Narrow window in parameter space where rabies can persist -->
<p>Overall, we find that accounting for both the spatial scale of vaccination and of mixing is critical to capturing the observed dynamics of canine rabies in the Serengeti District. Confirming previous work, we show that modeling transmission at the scale of dispersal (~ 1km) is key to capturing stable endemic dynamics and also result in parameter estimates closest to empirical estimates from individual scale data. We also found that introductions of cases from outside the district are crucial to the maintenance of transmission. Importantly, spatial and spatiotemporal features of the data were key to differentiating between models and identifying parameter estimates, and the time series statistics of the data were generally the least predictive of model separation or parameter estimates. We find that high spatial coverage of vaccination campaigns is key to predicting control outcomes, even at a coarse administrative level. Here, we define <span class="math inline">\(R_{0}\)</span> as the mean of the offspring distribution, but not as a critical threshold for persistence. Our results suggest that moving away from simplistic herd immunity thresholds to more nuanced targets, (i.e. 70% of dogs in at least 70% of villages), may better predict control outcomes. In addition, given the importance of transmission heterogeneity and introductions in transmission, control measures targeted at these key features, i.e. coordinating control across larger geographic scales, could compound the dividends gained through vaccination alone.</p>
<p>We use a uniquely comprehensive data set in both temporal and spatial scope in our analysis. This data allow us to pin down key features and expectations of rabies transmission, and the vaccination and population data provide a lens into the underlying susceptible dynamics of the population. In addition, we have a strong empirical understanding of key epidemiological parameters governing rabies transmission (i.e. incubation and infectious periods, dispersal kernels). Regardless of working with strong priors and in a narrow parameter space, certain parameters were still not identifiable, and no model was consistently able to predict the data observed. The dispersion parameter, or the heterogeneity in the transmission process was largely unidentifiable given our prior estimate, although it tended to be higher than estimates from empirical data, which could reflect issues of model misspecification, but could also result from biases in empirical data and direct estimates of this parameter <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-susswein2020" role="doc-biblioref">Susswein and Bansal 2020</a>)</span>.</p>
<p>Further refining summary statistics that draw the most information out of the available data may be a way forward to deal with these identifiability issues. However, given the stochastic nature of transmission dynamics, focusing on whether the observed data falls in the general trajectory space of the model may be the best we can do. The RF-ABC method is limited to estimation of independent parameter estimates, and we use a simple approach of filtering parameter sets where all weights are non-zero to approximate joint distributions. However, more work is needed to validate this approach, and other inference tools such as Sequential Monte Carlo methods could be used to estimate parameters jointly.</p>
<p>There are also uncertainties in the data that we did not account for. We use coarse demographic estimates at the village scale to simulate populations and do not account for immigration, emigration, or colonization of new patches. Our coverage estimates could be sensitive to these assumptions, although at the village scale, they generally track with estimates of vaccination coverage from post-vaccination surveys. In addition, we do not account for uncertainty in the case data in terms of timing and location of cases, and we use data on probable rather than laboratory confirmed rabies cases, although this approach has been shown to be highly specific in identifying genuine rabies cases <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-Hampson2009" role="doc-biblioref">Hampson et al. 2009b</a>)</span>. We assume that case detection is high over this period, with approximately 85% of cases detected each month. More work is needed to assess how sensitive our parameter estimates and model choice are to detection scenarios. Finally, our simulated control and vaccination scenarios are phenomenological and simplified approximations, rather than explicit representations of control programs.</p>
<p>Overall, we excluded many features of transmission included to explain dynamics in previous rabies modeling studies. We simulate secondary cases rather than biting rates and exposure probabilities <span class="citation">(<a href="introduction.html#ref-Mancyinprep" role="doc-biblioref">Mancy et al., n.d.</a>)</span>, we do not account for human mediated dog movement within the district <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-ferguson2015" role="doc-biblioref">Ferguson et al. 2015</a>; <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-townsend2013" role="doc-biblioref">Sunny E. Townsend et al. 2013</a>)</span>, we do not model the wildlife population <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-fitzpatrick2012" role="doc-biblioref">Fitzpatrick et al. 2012</a>)</span>, and finally we do not include reactive control to rabies outbreaks <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-ferguson2015" role="doc-biblioref">Ferguson et al. 2015</a>; <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-townsend2013" role="doc-biblioref">Sunny E. Townsend et al. 2013</a>)</span>. However, many of these features are likely captured implicitly within this model. Human-mediated dog movements and wildlife cases may be captured in the introduction rate, and reactive control (that is human responses that are random rather than ones that scale with incidence) is likely captured in the dispersion parameter (i.e. tying/killing/isolation of dogs resulting in failed transmission events). Here, we show that including these features explicitly may not be necessary when modeling transmission at the appropriate spatial scale and when incorporating key epidemiological features of the data such as transmission heterogeneity. Reactive control in particular, has often been used as a mechanism in previous models to generate realistic endemic dynamics and prevent host population declines, however there is little evidence that these measures scale temporally with incidence <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-Hampson2009" role="doc-biblioref">Hampson et al. 2009b</a>)</span>. As the village level model did result in simulated trajectories matching our observed data, modeling dynamics at the scale of the control intervention may be sufficient to capture dynamics, and metapopulation models at the scale of control that approximate mixing could greatly reduce data barriers in implementing spatial models <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-pascual2011" role="doc-biblioref">Pascual, Roy, and Laneri 2011</a>)</span>.</p>
<p>This work broadly confirms previous work on canine rabies transmission dynamics showing the importance of coverage heterogeneity in determining control outcomes and the spatial scale of mixing <span class="citation">(<a href="introduction.html#ref-Mancyinprep" role="doc-biblioref">Mancy et al., n.d.</a>)</span>. Recent phylodynamic analyses of integrated sequence and epidemiological data have also highlighted the role of introductions from surrounding populations <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-zinsstag2017" role="doc-biblioref">Zinsstag et al. 2017</a>; <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-bourhy2016" role="doc-biblioref">Bourhy et al. 2016</a>; <a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-laager2019" role="doc-biblioref">Laager et al. 2019</a>)</span>. These studies were largely at the scale of urban cities, and the authors argue that this indicates that rabies is not maintained within the population being studied. We find similar results in the Serengeti context, a less densely populated mosaic of agropastoralist communities. Together this work suggests that maintenance for canine rabies likely occurs across large spatial scales <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-hampson2007" role="doc-biblioref">Hampson et al. 2007</a>)</span>, and that low frequency events such as long-tailed disease induced dispersal or human mediated dog movement could facilitate this <span class="citation">(<a href="introduction.html#ref-brunker2015" role="doc-biblioref">Brunker et al. 2015</a>; <a href="introduction.html#ref-talbi2010" role="doc-biblioref">Talbi et al. 2010</a>)</span>. This has significant implications for control in that there are unlikely to be key sources or sink populations, or a critical community size for persistence <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-keeling1997" role="doc-biblioref">Keeling 1997</a>)</span>. Rather, as phylogenetic analyses of cases from this district show, there are likely critical landscape structures over which rabies can persist <span class="citation">(<a href="spatial-scale-of-control-and-mixing-predicts-dynamics-of-canine-rabies.html#ref-brunker2018" role="doc-biblioref">Brunker et al. 2018</a>)</span>.</p>
<p>Broadly, these results indicate the need to integrate key epidemiological features of transmission into dynamic models. We show that many aspects of transmission that have been included in previous models can be incorporated implicitly, and that modeling both the spatial scale of control and transmission is important for capturing observed dynamics under vaccination and stable endemic dynamics. Practically, developing spatial control targets may be a useful approach to improving the link between targets and outcomes and providing improved guidelines for vaccine implementation. However, more work should be done on how landscape and community structure impacts these outcomes. Moving forward, our work highlights the importance of confronting models with data and of incorporating key epidemiological features into models to generate realistic dynamics across the control timeline.</p>
</div>
<div id="references-4" class="section level2" number="5.5">
<h2><span class="header-section-number">5.5</span> References</h2>
<div id="refs_main" class="references csl-bib-body hanging-indent">
<div id="ref-abela-ridder2016" class="csl-entry">
Abela-Ridder, B, L Knopf, S Martin, L Taylor, Gregorio Torres, and Katinka de Balogh. 2016. <span>“2016: The Beginning of the End of Rabies?”</span> <em>The Lancet Global Health</em>.
</div>
<div id="ref-bourhy2016" class="csl-entry">
Bourhy, Hervé, Emmanuel Nakouné, Matthew Hall, Pierre Nouvellet, Anthony Lepelletier, Chiraz Talbi, Laurence Watier, et al. 2016. <span>“Revealing the Micro-Scale Signature of Endemic Zoonotic Disease Transmission in an African Urban Setting.”</span> <em>PLoS Pathogens</em>.
</div>
<div id="ref-brunker2018" class="csl-entry">
Brunker, Kirstyn, Philippe Lemey, Denise A Marston, Anthony R Fooks, Ahmed Lugelo, Chanasa Ngeleja, Katie Hampson, and Roman Biek. 2018. <span>“Landscape Attributes Governing Local Transmission of an Endemic Zoonosis: Rabies Virus in Domestic Dogs.”</span> <em>Molecular Ecology</em>.
</div>
<div id="ref-brunker2015" class="csl-entry">
Brunker, Kirstyn, Denise A Marston, Daniel L Horton, Sarah Cleaveland, Anthony R Fooks, Rudovick Kazwala, Chanasa Ngeleja, et al. 2015. <span>“Elucidating the Phylodynamics of Endemic Rabies Virus in Eastern Africa Using Whole-Genome Sequencing.”</span> <em>Virus Evolution</em>.
</div>
<div id="ref-czupryna2016" class="csl-entry">
Czupryna, Anna M, Joel S Brown, Machunde A Bigambo, Christopher J Whelan, Supriya D Mehta, Rachel M Santymire, Felix J Lankester, and Lisa J Faust. 2016. <span>“Ecology and Demography of Free-Roaming Domestic Dogs in Rural Villages Near Serengeti National Park in Tanzania.”</span> <em>PLoS ONE</em>.
</div>
<div id="ref-R-data.table" class="csl-entry">
Dowle, Matt, and Arun Srinivasan. 2020. <em>Data.table: Extension of ‘Data.frame‘</em>. <a href="https://CRAN.R-project.org/package=data.table">https://CRAN.R-project.org/package=data.table</a>.
</div>
<div id="ref-ferguson2015" class="csl-entry">
Ferguson, Elaine A, Katie Hampson, Sarah Cleaveland, Ramona Consunji, Raffy Deray, John Friar, Daniel T Haydon, Joji Jimenez, Marlon Pancipane, and Sunny E Townsend. 2015. <span>“Heterogeneity in the Spread and Control of Infectious Disease: Consequences for the Elimination of Canine Rabies.”</span> <em>Nature Publishing Group</em>.
</div>
<div id="ref-fitzpatrick2012" class="csl-entry">
Fitzpatrick, Meagan C, Katie Hampson, Sarah Cleaveland, Lauren Ancel Meyers, Jeffrey P Townsend, and Alison P Galvani. 2012. <span>“Potential for Rabies Control Through Dog Vaccination in Wildlife-Abundant Communities of Tanzania.”</span> <em>PLoS Neglected Tropical Diseases</em>.
</div>
<div id="ref-funk2020" class="csl-entry">
Funk, Sebastian, and Aaron A. King. 2020. <span>“Choices and Trade-Offs in Inference with Infectious Disease Models.”</span> <em>Epidemics</em> 30 (March): 100383. <a href="https://doi.org/10.1016/j.epidem.2019.100383">https://doi.org/10.1016/j.epidem.2019.100383</a>.
</div>
<div id="ref-hampson2015" class="csl-entry">
Hampson, Katie, Laurent Coudeville, Tiziana Lembo, Maganga Sambo, Alexia Kieffer, Michaël Attlan, Jacques Barrat, et al. 2015. <span>“Estimating the Global Burden of Endemic Canine Rabies.”</span> <em>PLoS Neglected Tropical Diseases</em>.
</div>
<div id="ref-hampson2007" class="csl-entry">
Hampson, Katie, Jonathan Dushoff, John Bingham, Gideon Brückner, Y H Ali, and Andy Dobson. 2007. <span>“Synchronous Cycles of Domestic Dog Rabies in Sub-Saharan Africa and the Impact of Control Efforts.”</span> <em>Proceedings of the National Academy of Sciences</em>.
</div>
<div id="ref-Hampson2009" class="csl-entry">
Hampson, Katie, Jonathan Dushoff, Sarah Cleaveland, Daniel T Haydon, Magai Kaare, Craig Packer, and Andy Dobson. 2009. <span>“Transmission Dynamics and Prospects for the Elimination of Canine Rabies.”</span> Edited by Charles E Rupprecht. <em>PLoS Biology</em> 7 (3): e1000053. <a href="https://doi.org/10.1371/journal.pbio.1000053">https://doi.org/10.1371/journal.pbio.1000053</a>.
</div>
<div id="ref-hazelbag2020" class="csl-entry">
Hazelbag, C. Marijn, Jonathan Dushoff, Emanuel M. Dominic, Zinhle E. Mthombothi, and Wim Delva. 2020. <span>“Calibration of Individual-Based Models to Epidemiological Data: A Systematic Review.”</span> Edited by Roger Dimitri Kouyos. <em>PLOS Computational Biology</em> 16 (5): e1007893. <a href="https://doi.org/10.1371/journal.pcbi.1007893">https://doi.org/10.1371/journal.pcbi.1007893</a>.
</div>
<div id="ref-R-raster" class="csl-entry">
Hijmans, Robert J. 2020. <em>Raster: Geographic Data Analysis and Modeling</em>. <a href="https://rspatial.org/raster">https://rspatial.org/raster</a>.
</div>
<div id="ref-jordan2019" class="csl-entry">
Jordan, Alexander, Fabian Krüger, and Sebastian Lerch. 2019. <span>“Evaluating Probabilistic Forecasts with scoringRules.”</span> <em>Journal of Statistical Software</em> 90 (12). <a href="https://doi.org/10.18637/jss.v090.i12">https://doi.org/10.18637/jss.v090.i12</a>.
</div>
<div id="ref-juul2020" class="csl-entry">
Juul, Jonas L., Kaare Græsbøll, Lasse Engbo Christiansen, and Sune Lehmann. 2020. <span>“Fixed-Time Descriptive Statistics Underestimate Extremes of Epidemic Curve Ensembles.”</span> <em>Nature Physics</em> 17 (1): 5–8. <a href="https://doi.org/10.1038/s41567-020-01121-y">https://doi.org/10.1038/s41567-020-01121-y</a>.
</div>
<div id="ref-Kain2020" class="csl-entry">
Kain, Morgan P., Marissa L. Childs, Alexander D. Becker, and Erin A. Mordecai. 2020. <span>“Chopping the Tail: How Preventing Superspreading Can Help to Maintain COVID-19 Control.”</span> <a href="https://doi.org/10.1101/2020.06.30.20143115">https://doi.org/10.1101/2020.06.30.20143115</a>.
</div>
<div id="ref-keeling1997" class="csl-entry">
Keeling, M. J. 1997. <span>“Disease Extinction and Community Size: Modeling the Persistence of Measles.”</span> <em>Science</em> 275 (5296): 65–67. <a href="https://doi.org/10.1126/science.275.5296.65">https://doi.org/10.1126/science.275.5296.65</a>.
</div>
<div id="ref-laager2019" class="csl-entry">
Laager, Mirjam, Monique Léchenne, Kemdongarti Naissengar, Rolande Mindekem, Assandi Oussiguere, Jakob Zinsstag, and Nakul Chitnis. 2019. <span>“A Metapopulation Model of Dog Rabies Transmission in n’djamena, Chad.”</span> <em>Journal of Theoretical Biology</em>.
</div>
<div id="ref-lankester2014" class="csl-entry">
Lankester, Felix, Katie Hampson, Tiziana Lembo, Guy Palmer, Louise Taylor, and Sarah Cleaveland. 2014. <span>“Implementing Pasteur’s Vision for Rabies Elimination.”</span> <em>Science</em>.
</div>
<div id="ref-Mancyinprep" class="csl-entry">
Mancy, Rebecca, Daniel Haydon, Ahmed Lugelo, Brunker Kirstyn, Sarah Cleaveland, Elaine Ferguson, Hotopp Karen, et al. n.d. <span>“Spatial Scale of Transmission Explains How Acute Infections Circulate at Low Prevalence.”</span>
</div>
<div id="ref-pascual2011" class="csl-entry">
Pascual, Mercedes, Manojit Roy, and Karina Laneri. 2011. <span>“Simple Models for Complex Systems: Exploiting the Relationship Between Local and Global Densities.”</span> <em>Theoretical Ecology</em>.
</div>
<div id="ref-sf2018" class="csl-entry">
Pebesma, Edzer. 2018. <span>“<span class="nocase">Simple Features for R: Standardized Support for Spatial Vector Data</span>.”</span> <em><span>The R Journal</span></em> 10 (1): 439–46. <a href="https://doi.org/10.32614/RJ-2018-009">https://doi.org/10.32614/RJ-2018-009</a>.
</div>
<div id="ref-R-sf" class="csl-entry">
———. 2021. <em>Sf: Simple Features for r</em>. <a href="https://CRAN.R-project.org/package=sf">https://CRAN.R-project.org/package=sf</a>.
</div>
<div id="ref-pudlo2015" class="csl-entry">
Pudlo, Pierre, Jean-Michel Marin, Arnaud Estoup, Jean-Marie Cornuet, Mathieu Gautier, and Christian P. Robert. 2015. <span>“Reliable ABC Model Choice via Random Forests.”</span> <em>Bioinformatics</em> 32 (6): 859–66. <a href="https://doi.org/10.1093/bioinformatics/btv684">https://doi.org/10.1093/bioinformatics/btv684</a>.
</div>
<div id="ref-R-program" class="csl-entry">
R Core Team. 2020. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="https://www.R-project.org/">https://www.R-project.org/</a>.
</div>
<div id="ref-rajeev2020modeling" class="csl-entry">
Rajeev, Malavika, C Jessica E Metcalf, and Katie Hampson. 2020. <span>“Modeling Canine Rabies Virus Transmission Dynamics.”</span> In <em>Rabies: 4th Edition: Scientific Basis of the Disease and Its Management</em>.
</div>
<div id="ref-raynal2018" class="csl-entry">
Raynal, Louis, Jean-Michel Marin, Pierre Pudlo, Mathieu Ribatet, Christian P Robert, and Arnaud Estoup. 2018. <span>“ABC Random Forests for Bayesian Parameter Inference.”</span> Edited by Oliver Stegle. <em>Bioinformatics</em> 35 (10): 1720–28. <a href="https://doi.org/10.1093/bioinformatics/bty867">https://doi.org/10.1093/bioinformatics/bty867</a>.
</div>
<div id="ref-R-foreach" class="csl-entry">
Revolution Analytics, and Steve Weston. n.d. <em>Foreach: Provides Foreach Looping Construct</em>.
</div>
<div id="ref-scott2017" class="csl-entry">
Scott, Terence P, Andre Coetzer, Anna S Fahrion, and Louis H Nel. 2017. <span>“Addressing the Disconnect Between the Estimated, Reported, and True Rabies Data: The Development of a Regional African Rabies Bulletin.”</span> <em>Frontiers in Veterinary Science</em>.
</div>
<div id="ref-susswein2020" class="csl-entry">
Susswein, Zachary, and Shweta Bansal. 2020. <span>“Characterizing Superspreading of SARS-CoV-2 : From Mechanism to Measurement.”</span> <a href="https://doi.org/10.1101/2020.12.08.20246082">https://doi.org/10.1101/2020.12.08.20246082</a>.
</div>
<div id="ref-talbi2010" class="csl-entry">
Talbi, Chiraz, Philippe Lemey, Marc A Suchard, Elbia Abdelatif, Mehdi Elharrak, Nourlil Jalal, Abdellah Faouzi, et al. 2010. <span>“Phylodynamics and Human-Mediated Dispersal of a Zoonotic Virus.”</span> <em>PLoS Pathogens</em>.
</div>
<div id="ref-townsend2013a" class="csl-entry">
Townsend, Sunny E., Tiziana Lembo, Sarah Cleaveland, François X. Meslin, Mary Elizabeth Miranda, Anak Agung Gde Putra, Daniel T. Haydon, and Katie Hampson. 2013. <span>“Surveillance Guidelines for Disease Elimination: A Case Study of Canine Rabies.”</span> <em>Comparative Immunology, Microbiology and Infectious Diseases</em> 36 (3): 249–61. <a href="https://doi.org/10.1016/j.cimid.2012.10.008">https://doi.org/10.1016/j.cimid.2012.10.008</a>.
</div>
<div id="ref-townsend2013" class="csl-entry">
Townsend, Sunny E, I Putu Sumantra, Pudjiatmoko, Gusti Ngurah Bagus, Eric Brum, Sarah Cleaveland, Sally Crafter, et al. 2013. <span>“Designing Programs for Eliminating Canine Rabies from Islands: Bali, Indonesia as a Case Study.”</span> <em>PLoS Neglected Tropical Diseases</em>.
</div>
<div id="ref-R-ranger" class="csl-entry">
Wright, Marvin N., Stefan Wager, and Philipp Probst. 2020. <em>Ranger: A Fast Implementation of Random Forests</em>. <a href="https://github.com/imbs-hl/ranger">https://github.com/imbs-hl/ranger</a>.
</div>
<div id="ref-ranger2017" class="csl-entry">
Wright, Marvin N., and Andreas Ziegler. 2017. <span>“<span class="nocase">ranger</span>: A Fast Implementation of Random Forests for High Dimensional Data in <span>C++</span> and <span>R</span>.”</span> <em>Journal of Statistical Software</em> 77 (1): 1–17. <a href="https://doi.org/10.18637/jss.v077.i01">https://doi.org/10.18637/jss.v077.i01</a>.
</div>
<div id="ref-zinsstag2017" class="csl-entry">
Zinsstag, Jakob, Monique Léchenne, Mirjam Laager, Rolande Mindekem, Service Naïssengar, Assandi Oussiguere, Kebkiba Bidjeh, et al. 2017. <span>“Vaccination of Dogs in an African City Interrupts Rabies Transmission and Reduces Human Exposure.”</span> <em>Science Translational Medicine</em>.
</div>
</div>
<div style="page-break-after: always;"></div>
</div>
<div id="supplementary-figures-1" class="section level2" number="5.6">
<h2><span class="header-section-number">5.6</span> Supplementary Figures</h2>
<div class="figure"><span id="fig:sfig-mod-ranks"></span>
<img src="figs/ch4/sfig_mod_ranks.jpeg" alt="Posterior probability of each model given our observed data, with the panels comparing models with fixed vs. estimated introductions, the colors showing the model scale, and the x-axis how movement was specified (see Methods)." width="90%" />
<p class="caption">
Figure 5.7: Posterior probability of each model given our observed data, with the panels comparing models with fixed vs. estimated introductions, the colors showing the model scale, and the x-axis how movement was specified (see Methods).
</p>
</div>

<div class="figure"><span id="fig:sfig-mod-vimp"></span>
<img src="figs/ch4/sfig_mod_vimp.jpeg" alt="Variable importance plots for models run with the full simulation data set (N = 1e5 simulations per model), or subsampled datasets (approx. 75% of the full simulation data set)." width="90%" />
<p class="caption">
Figure 5.8: Variable importance plots for models run with the full simulation data set (N = 1e5 simulations per model), or subsampled datasets (approx. 75% of the full simulation data set).
</p>
</div>

<div class="figure"><span id="fig:sfig-mod-err"></span>
<img src="figs/ch4/sfig_mod_err.jpeg" alt="Out-of-bag predictions of the out-of-bag error rate as trees are added separated by the scale of the model comparison." width="90%" />
<p class="caption">
Figure 5.9: Out-of-bag predictions of the out-of-bag error rate as trees are added separated by the scale of the model comparison.
</p>
</div>

<div class="figure"><span id="fig:sfig-posts-vimp"></span>
<img src="figs/ch4/sfig_posts_vimpgr.jpeg" alt="Variable importance plots for models run with the full simulation data set (N = 1e5 simulations per model), or subsampled datasets (approx. 75% of the full simulation data set) for each parameter and model." width="90%" />
<p class="caption">
Figure 5.10: Variable importance plots for models run with the full simulation data set (N = 1e5 simulations per model), or subsampled datasets (approx. 75% of the full simulation data set) for each parameter and model.
</p>
</div>

<div class="figure"><span id="fig:sfig-posts-jntvsfull"></span>
<img src="figs/ch4/sfig_posts_jntvsfull.jpeg" alt="Independent posterior estimates for models run with the full simulation data set (N = 1e5 simulations per model), or subsampled datasets (approx. 75% of the full simulation data set) for each parameter and model" width="90%" />
<p class="caption">
Figure 5.11: Independent posterior estimates for models run with the full simulation data set (N = 1e5 simulations per model), or subsampled datasets (approx. 75% of the full simulation data set) for each parameter and model
</p>
</div>

<div class="figure"><span id="fig:sfig-posts-err"></span>
<img src="figs/ch4/sfig_posts_err.jpeg" alt="Out-of-bag error rates as trees are added separated by the scale of the model comparison and the parameter being estimated." width="90%" />
<p class="caption">
Figure 5.12: Out-of-bag error rates as trees are added separated by the scale of the model comparison and the parameter being estimated.
</p>
</div>

<div class="figure"><span id="fig:sfig-par-recov"></span>
<img src="figs/ch4/sfig_param_recov.jpeg" alt="Simulated (true) parameter values vs. model estimates for 1000 simulated datasets witheld from the classification model." width="90%" />
<p class="caption">
Figure 5.13: Simulated (true) parameter values vs. model estimates for 1000 simulated datasets witheld from the classification model.
</p>
</div>

<div class="figure"><span id="fig:sfig-sims-full"></span>
<img src="figs/ch4/sfig_sims_full.jpeg" alt="Simulations from the independent posterior estimates for all the models. The grey envelope shows the range of simulations, and the black line is the time series of observed monthly cases. The orange lines show the top five simulations that best fit the data (lowest RMSE) and the purple lines show the top five simulations that have the highest centrality score." width="90%" />
<p class="caption">
Figure 5.14: Simulations from the independent posterior estimates for all the models. The grey envelope shows the range of simulations, and the black line is the time series of observed monthly cases. The orange lines show the top five simulations that best fit the data (lowest RMSE) and the purple lines show the top five simulations that have the highest centrality score.
</p>
</div>

<div class="figure"><span id="fig:sfig-sims-joint"></span>
<img src="figs/ch4/sfig_sims_joint.jpeg" alt="Simulations from the joint posterior estimates for all the models. The grey envelope shows the range of simulations, and the black line is the time series of observed monthly cases. The orange lines show the top five simulations that best fit the data (lowest RMSE) and the purple lines show the top five simulations that have the highest centrality score." width="90%" />
<p class="caption">
Figure 5.15: Simulations from the joint posterior estimates for all the models. The grey envelope shows the range of simulations, and the black line is the time series of observed monthly cases. The orange lines show the top five simulations that best fit the data (lowest RMSE) and the purple lines show the top five simulations that have the highest centrality score.
</p>
</div>

<div class="figure"><span id="fig:sfig-sims-crps"></span>
<img src="figs/ch4/sfig_sims_crps.jpeg" alt="Continuous ranked probability scores for the simulated datasets compared to the observed comparing models with estimated introduction rates (columns) and across scales (rows)." width="90%" />
<p class="caption">
Figure 5.16: Continuous ranked probability scores for the simulated datasets compared to the observed comparing models with estimated introduction rates (columns) and across scales (rows).
</p>
</div>

<div class="figure"><span id="fig:sfig-sims-endemic"></span>
<img src="figs/ch4/sfig_sims_endemic.jpeg" alt="The proportion of simulations resulting in population declines &gt; 15% by each time step for the best model at each scale." width="90%" />
<p class="caption">
Figure 5.17: The proportion of simulations resulting in population declines &gt; 15% by each time step for the best model at each scale.
</p>
</div>

<div class="figure"><span id="fig:sfig-conn-constraint"></span>
<img src="figs/ch4/sfig_conn_constraint.jpeg" alt="The relationship between district level proportion susceptible and reduction in introduction rate and limits on secondary cases." width="90%" />
<p class="caption">
Figure 5.18: The relationship between district level proportion susceptible and reduction in introduction rate and limits on secondary cases.
</p>
</div>

<div class="figure"><span id="fig:sfig-base-mets"></span>
<img src="figs/ch4/sfig_base_mets.jpeg" alt="Simulation outcomes across a range of campaign coverage and spatial coverage for the baseline case where panels are different summary statistics of the simulations (i.e. chain length mean, chain length peak, etc.)." width="90%" />
<p class="caption">
Figure 5.19: Simulation outcomes across a range of campaign coverage and spatial coverage for the baseline case where panels are different summary statistics of the simulations (i.e. chain length mean, chain length peak, etc.).
</p>
</div>

<div class="figure"><span id="fig:sfig-sspread-mets"></span>
<img src="figs/ch4/sfig_sspread_mets.jpeg" alt="Simulation outcomes across a range of campaign coverage and spatial coverage incorporating reductions in the maximum number of secondary cases rate as district coverage increases where panels are different summary statistics of the simulations (i.e. chain length mean, chain length peak, etc.)." width="90%" />
<p class="caption">
Figure 5.20: Simulation outcomes across a range of campaign coverage and spatial coverage incorporating reductions in the maximum number of secondary cases rate as district coverage increases where panels are different summary statistics of the simulations (i.e. chain length mean, chain length peak, etc.).
</p>
</div>

<div class="figure"><span id="fig:sfig-incs-mets"></span>
<img src="figs/ch4/sfig_incs_mets.jpeg" alt="Simulation outcomes across a range of campaign coverage and spatial coverage incorporating reductions in the introduction rate as district coverage increases where panels are different summary statistics of the simulations (i.e. chain length mean, chain length peak, etc.)." width="90%" />
<p class="caption">
Figure 5.21: Simulation outcomes across a range of campaign coverage and spatial coverage incorporating reductions in the introduction rate as district coverage increases where panels are different summary statistics of the simulations (i.e. chain length mean, chain length peak, etc.).
</p>
</div>

<div class="figure"><span id="fig:sfig-conn-mets"></span>
<img src="figs/ch4/sfig_conn_mets.jpeg" alt="Outbreak durations (weeks with cases &gt; 95th % of introductions) and their relationship to connectivity of villages across a range of campaign coverages (i.e. proportion vaccinated in each village, x-axis) and spatial coverage (proportion of villages vaccinated, panels)." width="90%" />
<p class="caption">
Figure 5.22: Outbreak durations (weeks with cases &gt; 95th % of introductions) and their relationship to connectivity of villages across a range of campaign coverages (i.e. proportion vaccinated in each village, x-axis) and spatial coverage (proportion of villages vaccinated, panels).
</p>
</div>


</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="modeling-canine-rabies-virus-transmission-dynamics.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="conclusion-1.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["dissertation.pdf"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
